tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -770,6 +770,11 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 			if err != nil {
 				return false, err
 			}
+			//validate tidbcluster status
+			if tc.Status.Monitor.Name != tm.Name || tc.Status.Monitor.Namespace != tm.Namespace {
```

Comment:
No need to add a check here, I would like to remove the status update to the TC in the future.